docs/agent-oriented-approach.md

```diff
@@ -0,0 +1,49 @@
+# Agent-oriented problem solving
+In this section, we highlight some of the most fundamental characteristics of the agent-based approach to solution development, and problem solving in general, which may be different from some of the existing paradigms and methodologies people are used to. We hope to be able to guide you towards having the right mindset when designing your own agent-based solutions to real world problems. 
+
+<!-- Our aim with the following discussion is that we guide you--> 
```

Comment:
Please remove comments..it is not good practice to keep comments in published work unless they are adding value.

Reply:
Will remove in the final version.